packages/label-sync/src/label-sync.ts

```diff
@@ -41,19 +41,20 @@ interface Repos {
 // from the local copy.  We are using the `PushEvent` to detect the change,
 // meaning the file running in cloud will be older than the one on master.
 let labelsCache: Labels;
-
-async function getLabels() {
+async function getLabels(github: GitHubAPI) {
   if (!labelsCache) {
-    await refreshLabels();
+    await refreshLabels(github);
   }
   return labelsCache;
 }
 
-async function refreshLabels() {
-  const url =
-    'https://github.com/googleapis/repo-automation-bots/blob/master/packages/label-sync/src/labels.json';
-  const res = await request<Labels>({ url });
-  labelsCache = res.data;
+async function refreshLabels(github: GitHubAPI) {
+  const data = (await github.repos.getContents({
```

Comment:
Oooops, my mistake.  I think you can fix this just by updating the url to https://raw.githubusercontent.com/googleapis/repo-automation-bots/master/packages/label-sync/src/labels.json

Shouldn't need to use the API and thread all this through for a simple GET (you know, when you use the right url 🤦‍♂ )

Reply:
One reason I'd advocate using the authenticated API instance we have available, is that it means that the request will fall under the quota of an authenticated client, vs., an unauthenticated request (which I believe has a significantly lower request limit).

Might not matter for the throughput of this bot, but worth considering.